components/ArtworksList/index.js

```diff
@@ -4,28 +4,36 @@ import styled from "styled-components";
 
 export default function ArtworksList({ artworks }) {
   return (
-    <>
-      <StyledList>
-        {artworks.items.map(({ id, image_thumbnail, titles }) => {
-          const titleText = titles[0].title;
-          return (
-            <li key={id}>
-              <Link href={`/artwork-info/${id}`}>
-                <Image
-                  src={image_thumbnail}
-                  alt={titleText}
-                  height={300}
-                  width={300}
-                />
-              </Link>
-            </li>
-          );
-        })}
-      </StyledList>
-    </>
+    <StyledList>
+      {artworks.items.map(({ id, image_thumbnail, titles }) => {
+        const titleText = titles[0].title;
+        return (
+          <StyledListItem key={id}>
+            <Link href={`/artwork-info/${id}`}>
+              <StyledImage
+                src={image_thumbnail}
+                alt={titleText}
+                width={200}
+                height={200}
+              />
+            </Link>
+          </StyledListItem>
+        );
+      })}
+    </StyledList>
   );
 }
 
 const StyledList = styled.ul`
   list-style-type: none;
+  padding: 0;
+`;
+
+const StyledImage = styled(Image)`
```

Comment:
I figured you used this name two times on different components (ArtworksList and ArtworksInfo). If you do your final styling later on, you might want to put both together and use a boolian or so (or use different naming)...But also not 100% sure about the common practice here